Dockerfile

```diff
@@ -66,6 +66,8 @@ RUN pg_ctl start ; \
 #
 USER root
 RUN mkdir /opt/odoo-addons
+RUN mkdir /opt/odoo-addons/addons
+RUN chmod -R 777 /opt/odoo-addons/
```

Comment:
There is no explanation for this change, nor does it seem to be related in any way to the subject of the pull request.  Please either explain or remove this commit.

Reply:
I've updated the commit message to explain the changes I made